### ISSUE TITLE
Why does the prompt on the right overflow to the next line?

### ISSUE THREAD

**Issue by fandean** [Powerlevel9k/powerlevel9k#1386]
I'm just moving vcs from the left to the right.

Why does the prompt on the right overflow to the next line?

![](https://i.loli.net/2019/12/08/FmRY8tPcz91nJjK.png)

#### Environment Information

  - windows 10 : wsl 
  - zsh --version : zsh 5.4.2 (x86_64-ubuntu-linux-gnu)
  - Oh-My-ZSH
  - install P9k (cloning the repo )
  - terminal emulator： Cmder

```
POWERLEVEL9K_PROMPT_ADD_NEWLINE=true

# LEFT_PROMPT
# ==============================
#POWERLEVEL9K_LEFT_PROMPT_ELEMENTS=(os_icon time dir dir_writable vcs)
POWERLEVEL9K_LEFT_PROMPT_ELEMENTS=( os_icon time dir dir_writable)
POWERLEVEL9K_PROMPT_ON_NEWLINE=true
POWERLEVEL9K_MULTILINE_FIRST_PROMPT_PREFIX=""
POWERLEVEL9K_MULTILINE_LAST_PROMPT_PREFIX=" $ "
POWERLEVEL9K_WHITESPACE_BETWEEN_LEFT_SEGMENTS=' '
POWERLEVEL9K_LEFT_SUBSEGMENT_SEPARATOR=' '

# RIGHT_PROMP
# ================================
POWERLEVEL9K_RIGHT_PROMPT_ELEMENTS=(vcs)
#POWERLEVEL9K_RIGHT_PROMPT_ELEMENTS=(status)

POWERLEVEL9K_MODE='nerdfont-complete'
ZSH_THEME='powerlevel9k/powerlevel9k'
```

**Comment by kuretru** [comment]
Same issue here.

**Comment by dgdosen** [comment]
I've got this issue to - and it's incredibly irritating

I get this issue when I use tmux and powerlevel9k. Sometimes, the bottom line of my neovim window pushes up, along with every other line in the vim window, and redraws that bottomline, so now there are two of them - and I need to quit... 

There seems to be an issue with powerlevel9k, coc and tmux all making changes to that part of the screen...

![Screen Shot 2020-01-29 at 1 22 45 PM](https://user-images.githubusercontent.com/75257/73398502-77c4bb80-429a-11ea-9ae1-18b5b6715af7.png)

Any ideas?

**Comment by romkatv** [comment]
@dgdosen Do you believe the problem you are experiencing is related to the issue you are commenting on? If so, why?

**Comment by romkatv** [comment]
Just to clarify, the original issue is simply about a bug in Cmder. Cmder is unable to properly display characters that are classified as "wide" in the Unicode standard. It has nothing to do with tmux, zsh, powerlevel10k or vim. Just a Cmder bug.

What you report seems something else, right?

**Comment by dgdosen** [comment]
It might be - I'm just getting the upward shifts in tmux/powerlevel9k.  If I'm editing a line in vim, my pointer stays in the same place, but  the underlying text shifts up as well...

**Comment by romkatv** [comment]
Pretty sure it's a different issue.

**Comment by dgdosen** [comment]
I agree - I think my issue was caused by using common dotfiiles across machines, and having powerlevel9k installed differently in each case (on macs)...

**Comment by jckling** [comment]
Same problem :(
